src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,24 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => {
+  if (comment) {
+    const { name, email, body } = comment;
+    const emailHref = `mailto:${email}`;
+
+    return (
+      <div className="CommentInfo">
+        <div className="CommentInfo__title">
+          <strong className="CommentInfo__name">{name}</strong>
+
+          {' by '}
+
+          <a className="CommentInfo__email" href={emailHref}>
+            {email}
+          </a>
+        </div>
+
+        <div className="CommentInfo__body">{body}</div>
+      </div>
+    );
+  }
+
+  return <></>;
+};
```

Comment:
You can check for the comments in the parant block, and just now to show this componenet, then to return <></> from it